com.github.PopoutApps.popout3d.yml

```diff
@@ -0,0 +1,36 @@
+app-id: com.github.PopoutApps.popout3d
+
+runtime: org.gnome.Platform
+runtime-version: "3.38"
+sdk: org.gnome.Sdk
+
+command: popout3d
+rename-desktop-file: popout3d.desktop
+rename-icon: popout3d
```

Comment:
```suggestion
rename-icon: popout3d_logo
```

Reply:
Logo - I was getting in a right mess with this. This used to work, but I removed the _logo from everything as I thought it was non-standard, I can't understand where it's coming from now. Commited as you suggested.

Have to stop work on this now.

Reply:
Yeah, these are not errors, just a message that flatpak found some icon but it doesn't match the ID.

Reply:
_...just a message that flatpak found some icon but it doesn't match the ID._
Didn't understand this, but:
_Yeah, these are not errors,_
This is what matters.